--- chroot.0/usr/share/initramfs-tools/scripts/live	2007-10-08 15:51:21.000000000 -0700
+++ chroot/usr/share/initramfs-tools/scripts/live	2007-10-16 16:23:25.000000000 -0700
@@ -1093,16 +1093,6 @@
 					return 0
 				fi
 			done
-		elif [ "${fstype}" = "squashfs" -o \
-			"${fstype}" = "ext2" -o \
-			"${fstype}" = "ext3" ]
-		then
-			# This is an ugly hack situation, the block device has
-			# an image directly on it.  It's hopefully
-			# live-initramfs, so take it and run with it.
-			ln -s "${devname}" "${devname}.${fstype}"
-			echo "${devname}.${fstype}"
-			return 0
 		fi
 	done
 
